.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm test
```

Comment:
also it would be nice to add https://github.com/allthings/cockpit/blob/master/.husky/commit-msg